docs/material/partials/header.html

```diff
@@ -8,7 +8,7 @@
           {% elif config.theme.logo.startswith("http") %}
             <img src="{{ config.theme.logo }}" width="24" height="24">
           {% else %}
-            <img src="{{ base_url }}/{{ config.theme.logo }}" width="153" height="15">
+            <img src="{{ base_url }}/{{ config.theme.logo }}" width="80">
```

Comment:
May be it is just me, but feels like alignment is a little odd and justified more to the left as compared to text `Delphix Virtualization SDK 4.0.0
<img width="580" alt="Screenshot 2022-04-08 at 8 29 47 AM" src="https://user-images.githubusercontent.com/45626610/162354527-6eaa4de1-174c-4181-9252-5e0821e337df.png">
`